sources/packages/web/src/services/http/dto/Common.dto.ts

```diff
@@ -65,3 +53,18 @@ export interface InstitutionPrimaryContactAPIOutDTO {
   primaryContactLastName: string;
   primaryContactPhone: string;
 }
+
+/**
+ * Common DTO for address details.
+ */ export interface AddressDetailsFormAPIDTO {
```

Comment:
Can you please move this to a new line ?

Reply:
mm, actually that is a line between 2 interfaces

Reply:
Is the DTO for in or out ?

Reply:
it's used in both IN and OUT, that's why I didn't specify IN or OUT. let me know devs thoughts

Reply:
This is supposed to reflect the API classes. If the API has only one we can keep only one and vice-versa.